<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLearn</class>
 <widget class="QWidget" name="DialogLearn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogLearn</string>
  </property>
  <widget class="QPushButton" name="btn_msg">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Message Dialog</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ColorBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Box</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>input Dialog</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
